.changeset/neat-avocados-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/app': patch
```

Comment:
@isaacroldan any concerns with making this a patch?

Reply:
nop, sounds good 👌